tests/e2e/specs/performance/cart.test.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Internal dependencies
+ */
+import { shopper, getLoadingDurations } from '../../../utils';
+import { SIMPLE_PHYSICAL_PRODUCT_NAME } from '../../../utils/constants';
+import { logPerformanceResult } from '../../utils';
+
+describe( 'Cart performance', () => {
+	beforeAll( async () => {
+		await shopper.goToShop();
+		await shopper.addToCartFromShopPage( SIMPLE_PHYSICAL_PRODUCT_NAME );
+	} );
+
+	it( 'Loading', async () => {
+		await shopper.block.goToCart();
+
+		const results = {
+			serverResponse: [],
+			firstPaint: [],
+			domContentLoaded: [],
+			loaded: [],
+			firstContentfulPaint: [],
+			firstBlock: [],
+			type: [],
+			focus: [],
+			inserterOpen: [],
+			inserterHover: [],
+			inserterSearch: [],
+			listViewOpen: [],
+		};
+
+		let i = 3;
+
+		// Measuring loading time.
+		while ( i-- ) {
+			await page.reload();
+			await page.waitForSelector( '.wc-block-cart', { timeout: 120000 } );
+			const {
+				serverResponse,
+				firstPaint,
+				domContentLoaded,
+				loaded,
+				firstContentfulPaint,
+				firstBlock,
+			} = await getLoadingDurations();
+
+			// Multiply by 1000 to get time in ms
+			results.serverResponse.push( serverResponse * 1000 );
+			results.firstPaint.push( firstPaint * 1000 );
+			results.domContentLoaded.push( domContentLoaded * 1000 );
+			results.loaded.push( loaded * 1000 );
+			results.firstContentfulPaint.push( firstContentfulPaint * 1000 );
+			results.firstBlock.push( firstBlock * 1000 );
+		}
+
+		Object.entries( results ).forEach( ( [ name, value ] ) => {
+			if (
+				Array.isArray( value ) &&
+				value.every( ( x ) => typeof x === 'number' ) &&
+				value.length === 0
+			) {
+				return;
+			}
+			logPerformanceResult( `Cart block loading: (${ name })`, value );
+		} );
+
+		// To stop warning about no assertions.
+		expect( true ).toBe( true );
+	} );
+
+	it( 'Quantity change', async () => {
+		await shopper.block.goToCart();
+		await page.waitForNetworkIdle( { idleTime: 2000 } );
+		await page.waitForSelector(
+			'button.wc-block-components-quantity-selector__button--plus'
+		);
+		let i = 10;
```

Comment:
I'd set this to 3, same as the previous test to 1) speed the test up and 2) to keep consistent data points throughout our tests